python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,6 +368,19 @@ def _check_getter(self):
             raise AttributeError('cannot modify or delete "%s" property (declared as "getter")' % _getName(self))
 
     def __get__(self, obj, cls):
+
+        self_name = _getName(self)
+        GangaObject._acquireChildLock(self_name, obj)
```

Comment:
I know and it should be an attribute which is assigned to a ```threading.Lock()``` but I'm still getting the error from passing a lock to a ```while myLockObject:``` line of code :/